packages/bbui/src/Table/Table.svelte

```diff
@@ -28,7 +28,7 @@
   export let rowCount = 0
   export let quiet = false
   export let loading = false
-  export let allowSelectRows = true
+  export let allowSelectRows
```

Comment:
Any reason in particular you changed the default value for this? We would just need to be careful there are no usages anywhere in the platform that depend on the existing default value.

Reply:
That was a separate bug fix, forgot to mention. This was a fix for tables that were defaulting to showing the selection checkboxes, despite the fact that 'Allow row selection' was not checked.

The manifest contains a default value of false.